pyscriptjs/src/remote_interpreter.ts

```diff
@@ -86,10 +86,9 @@ export class RemoteInterpreter extends Object {
      *
      * This is because, if it's used, loadPyodide
      * behaves mischievously i.e. it tries to load
-     * `pyodide.asm.js` and `pyodide_py.tar` but
-     * with paths that are wrong such as:
+     * additional files but with paths that are wrong such as:
      *
-     * http://127.0.0.1:8080/build/pyodide_py.tar
+     * http://127.0.0.1:8080/build/...
```

Comment:
Not really related to this PR, but this is what `indexURL` is for. But if you want to support multiple versions of Pyodide you can't vendor one version of `loadPyodide`. So probably the comment should say that we have to use the `loadPyodide` that came from the dynamically imported Pyodide to allow alternate Pyodide versions.

@antocuni was suggesting that we shouldn't even support multiple Pyodide versions, but that's yet another separate discussion I suppose.